js/.jscsrc

```diff
@@ -1,4 +1,5 @@
 {
+    "requireSemicolons": true,
```

Comment:
Additionally, I believe we had come to an agreement regarding the inconsistency of space usage in StandardJS in general